### PR TITLE
fix(): buildah exit on error

### LINKED ISSUE #38: buildah on error exit process
**Describe the bug**
Currently when buildah fails and execution, the error is logged with pty but the ocictl process is not exited.

**Expected behavior**
The process should execute on an internal buildah error

This fix is necessary for successfully completing the e2e testing suite.

### PR DESCRIPTION
**Pull request by artbegolli**
This is off of #37  - this is to be merged first.

closes #38

### CODE REVIEW

File: ocictl/cmd/build.go [hunk 1]
```diff
@@ -35,12 +35,13 @@ It can run a build in both docker and buildah varieties.
 `
 
 type buildCmd struct {
```
Comment (VaibhavPage) [inline ocictl/cmd/build.go]: can you run `go fmt`?

File: ocictl/cmd/build.go [hunk 2]
```diff
@@ -59,6 +60,7 @@ func newBuildCmd(out io.Writer) *cobra.Command {
 	f.StringVarP(&bc.builder, "builder", "b", "docker", "Choose either docker and buildah as the targetted image builder. By default the builder is docker.")
 	f.BoolVarP(&bc.debug, "debug", "d", false, "Turn on debug logging")
 	f.StringVarP(&bc.overlay, "overlay", "o", "", "Path to your overlay.yaml file")
+	f.StringVarP(&bc.storageDriver, "storage-driver", "s", "", "Storage-driver for Buildah. vfs enables the use of buildah within an unprivileged container. By default the storage driver is overlay")
```
Comment (VaibhavPage) [inline ocictl/cmd/build.go]: you may want to put `overlay` as default for the `&bc.storageDriver`

Comment (artbegolli) [reply ocictl/cmd/build.go]: Buildah does this but I'll add it to make it explicit

**Comment by VaibhavPage** [comment]
Conflicts

File: ocictl/cmd/build.go (hunk 1 above)
Comment (michael-bowen-sc) [reply ocictl/cmd/build.go]: We should run it as part of any local commit hook (husky?)

Comment (artbegolli) [reply ocictl/cmd/build.go]: created issue #56

**Pull request merged**